src/components/ChatApp/MessageListItem/Feedback.react.js

```diff
@@ -101,6 +88,17 @@ class Feedback extends React.Component {
   };
 
   render() {
+    const styles = {
```

Comment:
Define this styles constant outside the component declaration.

Reply:
Also, you can name the keys as  `buttonStyle` and so on, so that it makes sense after you destructure it.